app/controllers/users_controller.rb

```diff
@@ -0,0 +1,67 @@
+class UsersController < ApplicationController
+  # before_action :authenticate_user, except: %i[create signout]
+  before_action :set_user, only: %i[show update destroy]
+
+  # GET /users
+  def index
+    @users = User.all
+    render json: @users
+  end
+
+  # GET /users/1
+  def show
+    render json: @user
+  end
+
+  def logged_in?
```

Comment:
RESTful API's are supposed to be "stateless". This means your back end isn't responsible for keeping the authenticated state of any user. As such, this "logged_in" method isn't really useful. The back end's job as far as authenticating is concerned is simply to send out a token (which you are already doing) and then checking for the existence of that token in subsequent requests by the API consumer (aka the front end apps).

Besides, the action name "logged_in?" isn't RESTful and so doesn't belong to a controller

Reply:
True, I have instead decided to use user show method following rest convention

Reply:
I'm not sure I follow what you mean here but I still see the "logged_in?" method present. This particular method, as previously mentioned, doesn't actually add any value to the API as far as testing for user authentication is concerned. So we can safely remove this :)

Reply:
I have already removed logged_in method as requested in the last review. I checked the files changed and it is not there as well. I believe there has been a mistake. Please check the users controller again as I double checked and made sure that there indeed is no logged_in method anymore.

Reply:
My bad. Github didn't show it as removed in the diff

Reply:
No problem :+1: , Thanks